go-controller/pkg/ovn/gress_policy.go

```diff
@@ -73,8 +70,6 @@ func newGressPolicy(policyType knet.PolicyType, idx int, namespace, name string)
 		peerV4AddressSets: sets.String{},
 		peerV6AddressSets: sets.String{},
 		portPolicies:      make([]*portPolicy, 0),
-		ipBlockCidr:       make([]string, 0),
-		ipBlockExcept:     make([]string, 0),
```

Comment:
nit: why not make the slice here to be sure?

Reply:
you are guaranteed to call addIPBlock before generating the match string